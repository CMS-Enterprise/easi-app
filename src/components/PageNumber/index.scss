@use 'uswds-core' as *;

.easi-page-number {
  margin-top: 15px;

  &__page-num {
    color: color('base');
    font-size: 14px;
  }
}
