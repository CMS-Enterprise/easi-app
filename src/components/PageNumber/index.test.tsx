import React from 'react';
import { render } from '@testing-library/react';

import PageNumber from './index';

describe('The Page Number component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <PageNumber className="test-class-name" currentPage={2} totalPages={10} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
