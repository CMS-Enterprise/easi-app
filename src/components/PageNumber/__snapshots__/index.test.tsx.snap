// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`The Page Number component > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="easi-page-number test-class-name"
  >
    <span
      class="easi-page-number__page-num"
    >
      Page 2 of 10
    </span>
  </div>
</DocumentFragment>
`;
